src/components/Adyen/Adyen.js

```diff
@@ -288,17 +289,21 @@ const Adyen = ({
     const isBancontactCard =
       selectedPaymentMethodRef?.current?.methodName === 'bancontact_card';
 
-    let checkbox = document.querySelector(`.checkbox-${methodName}`);
+    let checkbox = document.querySelector(`#checkbox-${methodName}`);
+    let checkboxWrapper = document.querySelector(`.checkbox-${methodName}`);
 
     // condition below needs to be verified when new 'scheme' is added
     if (methodName === 'scheme') {
-      checkbox = document.querySelector(
-        `.checkbox-${isBancontactCard ? 'bcmc' : 'card'}`
+      const schemeCheckbox = document.querySelector(
+        `#checkbox-${isBancontactCard ? 'bcmc' : 'card'}`
       );
+
+      checkbox = schemeCheckbox;
+      checkboxWrapper = schemeCheckbox.parentElement;
```

Comment:
is it important what kind of a parent element it is? this might change in the future, and we won't know what happened

maybe some more specific selector would be better, like `.closest`?